pkgs/sdk_triage_bot/tool/bench.dart

```diff
@@ -114,8 +109,22 @@ class ClassificationResults {
   }
 
   bool satisfiedBy(List<String> labels) {
-    final filtered = labels.where((l) => !l.startsWith('type-')).toSet();
-    final expected = expectedLabels.where((l) => !l.startsWith('type-'));
-    return expected.every(filtered.contains);
+    // handle needs-info
+    if (expectedLabels.contains('needs-info')) {
```

Comment:
(I assume this is for testing/quality control only, so the repeated linear list lookups shouldn't be a problem. Right?)